pkg/model/trigger_mode.go

```diff
@@ -11,26 +11,29 @@ type TriggerMode int
 //      for an update
 const (
 	// Tilt automatically performs initial and non-initial builds without manual intervention
-	TriggerModeAuto TriggerMode = iota
+	TriggerModeAuto_AutoInit TriggerMode = iota
```

Comment:
i thought Golang names didn't have underscores, cf https://golang.org/doc/effective_go#mixed-caps

maybe call them

TriggerOnManualOnly
TriggerOnFileChange
TriggerOnInit
TriggerOnFileChangeOrInit

?

Reply:
I mean they're not supposed to have underscores, no, that's why I said i was cheating 😅

Reply:
hmmm...were you going to change the names?

Reply:
oh whoops i missed this. fixing now.